docs/sdk-and-tools/indices/accounts.md

```diff
@@ -0,0 +1,58 @@
+---
+id: es-index-accounts
+title: Elasticsearch index: accounts
+---
+
+The `_id` field for this index is composed of bech32 encoded address.
+
+| Field      | Description                                                                                                |
+|------------|------------------------------------------------------------------------------------------------------------|
+| address    | The address field holds the address bytes in a bech32 encoding.                                            |
+| balance    | The balance field holds the amount of EGLD the address has.                                                |
+| balanceNum | The balanceNum field holds the amount of EGLD the address has in a numeric format(denominated with 10^18). |
```

Comment:
The balance field holds the amount of EGLD the address possesses, in a numeric format. Example: `1.5`

Reply:
done